<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viettel-CA Token Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="loginMenuButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWidget" name="loginScreen" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="dataViewScreen" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="exportButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="viewButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="cerMenuButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWidget" name="settingMenu" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QPushButton" name="settingMenuPushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingMenuPushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="settingMenuLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="settingMenuButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>24</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QWidget" name="changePinScreen" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>281</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLabel" name="changePinlabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="changePinlabel_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="changePinlabel_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="changePinlabel_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="changePinlineEdit_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="changePinlineEdit_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="changePinlineEdit_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="changePinpushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changePinpushButton_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="changeNameScreen" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QPushButton" name="changeNamepushButton_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeNamepushButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="changeNamelineEdit_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="changeNamelabel_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="changeNamelabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <zorder>loginScreen</zorder>
   <zorder>dataViewScreen</zorder>
   <zorder>loginMenuButton</zorder>
   <zorder>cerMenuButton</zorder>
   <zorder>settingMenuButton</zorder>
   <zorder>settingMenu</zorder>
   <zorder>changeNameScreen</zorder>
   <zorder>changePinScreen</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>viewButton</tabstop>
  <tabstop>cerMenuButton</tabstop>
  <tabstop>settingMenuPushButton_2</tabstop>
  <tabstop>settingMenuPushButton</tabstop>
  <tabstop>settingMenuButton</tabstop>
  <tabstop>changePinlineEdit_1</tabstop>
  <tabstop>changePinlineEdit_2</tabstop>
  <tabstop>changePinlineEdit_3</tabstop>
  <tabstop>changePinpushButton_2</tabstop>
  <tabstop>changePinpushButton_1</tabstop>
  <tabstop>loginMenuButton</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>changeNamelineEdit_1</tabstop>
  <tabstop>changeNamepushButton_1</tabstop>
  <tabstop>changeNamepushButton_2</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
